code/game/machinery/suit_storage_unit.dm

```diff
@@ -778,6 +778,13 @@
 			to_chat(user, "You cannot refit a customised voidsuit.")
 			return
 
+		//VR ADDITION BEGINS

```

Comment:
I would recommend using //VOREStation Edit specifically (You can add more to it, but //VOREStation Edit is consistent and allows me to search for '//VOREStation' when searching for edits.

Reply:
Done.